src/heaplayers/heaps/top/mallocheap.h

```diff
@@ -27,7 +27,9 @@ extern "C" size_t malloc_usable_size (void *);
 #elif defined(__linux__)
 #include <malloc.h>
 #else
-extern "C" size_t malloc_usable_size (void *) throw ();
+extern "C" size_t
+malloc_usable_size (void *)
+throw ();
```

Comment:
```suggestion
extern "C" size_t malloc_usable_size (void *) throw ();
```
Modified during indentation?

Reply:
If you refer this macros and use it in src/base/memory_cwrapper.h, then I think this line also need to added in src/base/memory_cwrapper.h.

Reply:
The indentation result of git-hook can't make that. We have to add *INDENT-OFF* to make the function definition at one line in .h file(in .hpp or .cpp files, it doesn't make matter. So, what do you think? (one line vs. three lines)

Reply:
In other cpp and h files, it was written in a single line, so I'm wondering if there might be a mistake. For consistency, I think it is good to follow others (one line)

Reply:
OK. I change it.